service/resource/configmap/resource.go

```diff
@@ -188,7 +188,7 @@ func (r *Resource) GetDeleteState(ctx context.Context, obj, currentState, desire
 }
 
 func (r *Resource) GetUpdateState(ctx context.Context, obj, currentState, desiredState interface{}) (interface{}, interface{}, interface{}, error) {
-	return []*apiv1.ConfigMap{}, []*apiv1.ConfigMap{}, []*apiv1.ConfigMap{}, nil
+	return nil, nil, nil, nil
```

Comment:
I noticed the problem is not only concerned with the namespace resource but with all resources. The implications for the other resources are not that critical but also dangerous. So this fixes them.